src/common/mfu_flist_walk.c

```diff
@@ -666,7 +691,7 @@ void mfu_flist_walk_paths(uint64_t num_paths, const char** paths,
         if (time_diff > 0.0) {
             rate = ((double)all_count) / time_diff;
         }
-        MFU_LOG(MFU_LOG_INFO, "Walked %lu items in %f seconds (%f items/sec)",
+        MFU_LOG(MFU_LOG_INFO, "Walked %lu items in %f seconds (%f files/sec)",
```

Comment:
Let's keep this as items, since some objects in the count might be directories or symlinks in addition to "files".
```suggestion
        MFU_LOG(MFU_LOG_INFO, "Walked %lu items in %f seconds (%f items/sec)",
```